lib/proof/data/data_sources/rpc_proof_data_source.dart

```diff
@@ -9,45 +9,10 @@ import 'package:polygonid_flutter_sdk/sdk/di/injector.dart';
 import 'package:web3dart/crypto.dart';
 import 'package:web3dart/web3dart.dart';
 
-class RPCDataSource {
+class RPCProofDataSource {
```

Comment:
`RPCIdentityDataSource` and `RPCProofDataSource` look like duplicates?

Reply:
They are not, RPCIdentity calls the smart contract to get the identity state, and RPCProof calls the smart contract to get the gist (global identity state) proof, I'm still having doubts if GistProof should be in identity or in proof, for now I think its place is in proof part as it's a proof and we are using it for generating proofs also.